### PR TITLE
feat: [SRM-9975]: cypress service health dashboard

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:
Screenshot 2022-04-06 at 8 47 41 AM

Screenshot 2022-04-06 at 8 47 49 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: cypress/integration/85-cv/ServiceHealth/ServiceHealth.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    cy.contains('div', 'appd').click()
+    cy.wait('@monitoredServiceCall')
+    cy.contains('div', 'Service Health').click()
+    cy.wait('@heatlhScoreCall')
+    cy.wait('@heatlhScoreTimeLineCall')
+    cy.wait('@overAllHeatlhScoreCall')
+
+    cy.findByTestId('HealthScoreChartContainer').click()
+    cy.wait('@healthSourceListCall')
+    cy.wait('@metricsCall')
```
Comment (praneshg239) [inline cypress/integration/85-cv/ServiceHealth/ServiceHealth.spec.ts:43]: what we are asserting here?

Comment (deepesh-ui) [reply cypress/integration/85-cv/ServiceHealth/ServiceHealth.spec.ts:45]: Updated

File: src/modules/85-cv/components/HealthSourceDropDown/__tests__/HealthSourceDropDown.test.tsx [hunk 2]
```diff
@@ -98,4 +96,58 @@ describe('Unit tests for HealthSourceDropDown', () => {
...
+      .querySelector(`[name="healthsources-select"] + [class*="bp3-input-action"]`)
+      ?.querySelector('[data-icon="chevron-down"]')
+    await waitFor(() => {
+      fireEvent.click(selectCaretNewItem!)
+    })
+
+    const typeToSelectNewInput = await findByText(container, 'Appd Health source updated')
+    act(() => {
+      fireEvent.click(typeToSelectNewInput)
+    })
```
Comment (shashidhargg) [inline src/modules/85-cv/components/HealthSourceDropDown/__tests__/HealthSourceDropDown.test.tsx:151]: Are we missing any assertion after the click?

**Review by shashidhargg — APPROVED** [review]

**Review by shashidhargg — APPROVED** [review]

**Review by shashidhargg — APPROVED** [review]

**Review by shashidhargg — APPROVED** [review]

**Comment by deepesh-ui** [comment]
retrigger jest

**Comment by deepesh-ui** [comment]
retrigger coverage

**Comment by deepesh-ui** [comment]
retrigger jest

**Pull request merged**